core/src/init/toast/toast.ts

```diff
@@ -0,0 +1,19 @@
+import { ToastPane, text, toast, ToastPaneType } from "@moai/core";
+import { Check, Warning2 } from "@banhmi/icon/solid";
+import s from "./toast.module.css";
+ToastPane.types = {
```

Comment:
There should be a blank line between 3 and 4. Actually this should be covered in prettier..